### ISSUE TITLE
History jumps to the 1st entry on page refresh

### ISSUE THREAD

**Issue by itsalaidbacklife** [cuttle-cards/cuttle#183]
### Bug Summary

When you refresh the page inside a game, the history doesn’t scroll to the bottom entry, meaning you see the first move rather than the last move

### Detailed Description

We already have a feature to scroll the history to the latest entry that fires when a new move is made. We should abstract that scroll functionality into a method that gets called both when a new move is made and on page refresh

### Steps to Reproduce

1. Start a game
2. Play enough moves that the history scrolls
3. Refresh the page

**Comment by P-DR0ZD** [comment]
@itsalaidbacklife can I work on this

**Comment by itsalaidbacklife** [comment]
@P-DR0ZD definitely! Let me know if there's anything I can do to help you set up. Feel free to ask questions about the issue (or general setup) here in this issue thread, or if you'd like to chat in real-time about what you're working on, then try the #code channel of [our discord](https://discord.gg/6vQCFa6nvP)

**Comment by itsalaidbacklife** [comment]
I'll add that the relevant file is `client/js/views/GameView.vue` and that the watcher which scrolls upon new logs coming in is the only one in the watch block:

```
  watch: {
    logs: function () {
      this.$nextTick(function () {
        const container = this.$refs.logsContainer;
        if (container) {
          container.scrollTop = container.scrollHeight + 120;
        }
      });
    },
  },
```

**Comment by P-DR0ZD** [comment]
Thanks for the extra info. I'll get started on asap

**Comment by itsalaidbacklife** [comment]
As I look more closely at it, the existing watcher looks like it scrolls exactly the height of the container, meaning it'll only go to the 'next' page worth of content rather than always going to the bottom. It'll probably be best to jigger it to always scroll to the last item and re-use that here and on page load

**Comment by itsalaidbacklife** [comment]
Closed by #184
